tests/core/checks/custom/BUILD.bazel

```diff
@@ -0,0 +1,150 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//tests:bazel_tests.bzl", "bazel_test")
+
+_workspace = """
```

Comment:
Style nit: these variables can be `ALL_CAPS` since they are constants within a build file. The leading underscore is only needed in .bzl files to prevent variables from being imported by other files.

Reply:
Done.